server/src/main/java/org/elasticsearch/inference/ModelConfigurations.java

```diff
@@ -192,10 +225,14 @@ public XContentBuilder toFilteredXContent(XContentBuilder builder, Params params
         builder.field(TaskType.NAME, taskType.toString());
         builder.field(SERVICE, service);
         builder.field(SERVICE_SETTINGS, serviceSettings.getFilteredXContentObject());
-        builder.field(TASK_SETTINGS, taskSettings);
+        builder.field(OLD_TASK_SETTINGS, taskSettings);
         if (chunkingSettings != null) {
             builder.field(CHUNKING_SETTINGS, chunkingSettings);
         }
+        if (params.paramAsBoolean(INCLUDE_PARAMETERS, true)) { // default true so that REST requests get parameters
+            builder.field(PARAMETERS, taskSettings);
+        }
+        builder.field(ENDPOINT_VERSION_FIELD_NAME, endpointVersion);
```

Comment:
I guess maybe a counter point to this. As a user, I think there might be some confusion as to why some of my endpoints show `task_settings` while other show `parameters`. Showing this value will make it clear why this is the case since the documentation will presumably have to explain the various `endpoint_version`. As a related point, the naming `endpoint_version` might be a bit of a misnomer. I think it implies that your endpoint is versioned and may be confusing with the new update API. Would something like `endpoint_template_version` or `endpoint_schema_version` maybe be clearer?